components/skeleton/skeleton-mixin.js

```diff
@@ -20,7 +23,7 @@ export const skeletonStyles = css`
 		position: absolute;
 		right: 0;
 		top: 0;
-		z-index: 4;
+		z-index: 999;
```

Comment:
Checking I understand correctly - fixing the specific flyout issue is solved by setting the opacity and creating the stacking context.  This doesn't actually _need_ to raise here because `4` has worked so far (meaning everything implementing skeleton doesn't set its internal components that should skeletize to anything higher than that).  But because we have a stacking context now, we can set this to a much higher in case anything ever does?

Reply:
That's exactly my thinking!